src/rules/objectLiteralShorthandRule.ts

```diff
@@ -31,8 +31,8 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static LONGHAND_PROPERTY = "Expected property shorthand in object literal.";
-    public static LONGHAND_METHOD = "Expected method shorthand in object literal.";
+    public static LONGHAND_PROPERTY = "Expected property shorthand in object literal ('{foo, bar}').";
```

Comment:
I doubt all users will understand this, because it's not obvious that foo is a property name (it also looks syntactically like destructuring). Easy fix is print `{prop1, prop2}` instead so the names are self-documenting.
Ideally we should point to the https://palantir.github.io/tslint/rules/object-literal-shorthand/ page, and make that have more information explaining the syntax for newbies (they may not be aware of the shorthand syntax at all)